src/components/[guild]/RoleCard/components/Reward.tsx

```diff
@@ -217,6 +219,10 @@ const RewardIcon = ({
     transition: { type: "spring", duration: 0.5, ...transition },
   }
 
+  if (owerwriteImg) {
```

Comment:
don't we save the token image into guildPlatform.platformGuildData.imageUrl too, so this whole custom logic wouldn't be necessary?

Reply:
I believe this was accidentally left here, when I refactored the image saving part, thanks!